File: stage_2/main.go
package main

import "fmt"

func fibonacci(limit int) []int {
	sequence := []int{0, 1}
	for {
		next := sequence[len(sequence)-1] + sequence[len(sequence)-2]
		if next > limit {
			break
		}
		sequence = append(sequence, next)
	}
	return sequence
}

func main() {
	fib := fibonacci(100)
	fmt.Println("Ряд Фибоначчи до 100:")
	fmt.Println(fib)
}

File: stage_2/main.py
def fibonacci(limit):
    sequence = [0, 1]
    while True:
        next_num = sequence[-1] + sequence[-2]
        if next_num > limit:
            break
        sequence.append(next_num)
    return sequence

fib = fibonacci(100)
print("Ряд Фибоначчи до 100:")
print(fib)

